.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -293,10 +327,38 @@ stages:
             echo '##vso[task.setvariable variable=ENABLE_FIPS]false'
             echo '##vso[task.setvariable variable=ENABLE_TRUSTED_LAUNCH]False'
             echo '##vso[task.setvariable variable=SGX_INSTALL]True'
+            echo '##vso[task.setvariable variable=ENABLE_CGROUPV2]False'
           displayName: Setup Build Variables
         - template: ./templates/.builder-release-template.yaml
           parameters:
             artifactName: marinerv2-gen1
+  - stage: build_vhd_marinerv2_gen1_cgroupv2
```

Comment:
I guess I'm a bit confused, I thought we'd agreed on just plugging in `##vso[task.setvariable variable=ENABLE_CGROUPV2]True` into the existing marinerv2 SKUs which will then be shipped to the new AKSAzureLinnux RG/gallery

Reply:
Will this change add a new SKU? I thought this is where I will plug in ENABLE_CGROUPV2 to true to build the cgroupv2-falvored VHD.

Reply:
so we use the variables like `buildMarinerV2gen1` as parameters to specify we want to build specific SKUs by running the corresponding stages - it looks like you've added both new variables and corresponding stages to build independent SKUs for cgroupsv2. To just modify the existing mariner SKUs, you'd wanna only add the `cgroupsv2 = true` bit to the existing `*_marinerv2_*` stages, for example: https://github.com/Azure/AgentBaker/blob/a57fe0aa8b5011a081a877537e2200f213c02567/.pipelines/.vsts-vhd-builder-release.yaml#L274C1-L274C1

Reply:
But in that case, all the Mariner V2 images will be built as cgroupv2. We still want to build cgroupv1 Mariner V2 VHDs at this point.

Reply:
After offline discussion, agreement has been reached to add new SKU for the cgroupv2-flavored images.